### ISSUE TITLE
Step 7 of Snap the Dot tutorial hint doesn't show blocks

### ISSUE THREAD

**Issue by samelhusseini** [Microsoft/pxt-microbit#1650]
**Describe the bug**
Step 7 shows JS instead of Blocks.

**To Reproduce**
1. Go to the 'snap the dot' tutorial
2. Navigate to Step 7
3. Click the hint after Ok
3. See screenshot

**Screenshots**
<img width="1019" alt="screen shot 2018-11-18 at 8 34 59 am" src="https://user-images.githubusercontent.com/16690124/48675387-0ec9ab80-eb0d-11e8-819e-a3391ec26837.png">

**Desktop (please complete the following information):**
Chrome Mac

**Additional context**
Repro's on other browsers as well

**Comment by samelhusseini** [comment]
This is still not fixed. It might be that it's cached? or maybe something else is going on. Keeping open till we verify that it's fixed live.

**Comment by pelikhan** [comment]
Use anonymous window

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by samelhusseini** [comment]
Yeah still happening. I think it might be cached on crowdin? I know the md returned from the backend is cached. 

Here's what the live version is: 
https://makecode.microbit.org/api/md/microbit/projects/snap-the-dot

and here's what the editor is requesting. Notice how the last step is messed up in that verison. Not sure where it's coming from but I think crowdin. 

https://makecode.microbit.org/api/md/microbit/projects/snap-the-dot?targetVersion=1.2.11&lang=en

**Comment by pelikhan** [comment]
Bump to update crowdin

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by pelikhan** [comment]
We should not request crowdin when lang=en. Or client should not request it.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________
